src/components/ProbeList.tsx

```diff
@@ -54,7 +54,7 @@ export const ProbeList = ({ probes, onAddNew, onSelectProbe }: Props) => {
                 <span className="add-data-source-item-text">{probe.name}</span>
                 <span className="add-data-source-item-desc">
                   <Badge color={color} icon={onlineIcon} text={onlineTxt} />
-                  <div>{labelsToString(probe.labels)}</div>
+                  <div>{labelsToString(probe.labels).join(', ')}</div>
```

Comment:
[nit] this `.join()` could be chained onto the end of `labelsToString` if you want

Reply:
updated

Reply:
🔥